.gitignore

```diff
@@ -39,6 +39,10 @@ local.properties
 
 # OS specific files
 .DS_Store
+venv
 
 # Exclude gradle wrapper jar
 !gradle/wrapper/gradle-wrapper.jar
+
+# Test artifacts
+lib/src/test/resources
```

Comment:
I don't think we should be excluding all resources from git. We keep contracts used for tests there etc.

Reply:
Oh wow, I went over the top with this. There is a file that is being created as a result of running tests - `starknet_open_zeppelin_accounts.json` which I wanted to exclude from git. So I guess the question is, should I exclude it here explicitly OR we should maybe get rid of it after tests conclude?

Reply:
I think we should exclude only `lib/src/test/resources/account/` directory. Removing it after tests conclude can be done additionally, but I think that's for another PR.